pkg/registries/registry.go

```diff
@@ -181,3 +195,63 @@ func createFilter(config Config, log *logging.Logger) Filter {
 
 	return filter
 }
+
+func validateSpecs(log *logging.Logger, inSpecs []*apb.Spec) []*apb.Spec {
+	var wg sync.WaitGroup
+	wg.Add(len(inSpecs))
```

Comment:
I prefer to call the addition to the wait group as I call the go routines. To me, it makes it more clear what we are supposed to be waiting on. This is still really clear, just throwing it out there for the future.